src/hello/reducer.js

```diff
@@ -2,9 +2,9 @@
 
 import {combineReducers} from 'redux'
 
-import type {ReplyAction} from './types'
+import type {Action} from '../types'
 
-export const messageReducer = (state: string = '', {type, payload: message}: ReplyAction) => {
+export const messageReducer = (state: string = '', {type, payload: message}: $Subtype<Action>) => {
```

Comment:
The problem with specific action types in the `payload` property. What would be the best way to handle that, i.e. how should we define the `Action` type for it to work correctly?